includes/wp-cli.php

```diff
@@ -219,6 +219,68 @@ public function recount_indexed_posts() {
 	}
 
 	/**
+	 * @param $args
+	 * @param $assoc_args
+	 *
+	 * @subcommand cron
+	 */
+	public function cron( $args, $assoc_args ) {
+		$action = '';
+
+		if ( isset( $assoc_args['disable'] ) ) {
+			$action = 'disable';
+		} else if ( $assoc_args['enable'] ) {
+			$action = 'enable';
+		}
+
+		if ( empty( $action ) ) {
+			WP_CLI::error( 'specify a cron action - `--enable` or `--disable`' );
+		}
+
+		if ( method_exists( __CLASS__, $action . '_cron' ) ) {
+			$result      = call_user_func( array( __CLASS__, $action . '_cron' ) );
+			$action_past = $action . 'd';
+			$action_verb = trim( $action, 'e' ) . 'ing';
+
+			if ( $result ) {
+				WP_CLI::success( 'cron ' . $action_past );
+			} else {
+				WP_CLI::error( 'error ' . $action_verb . ' cron' );
+			}
+		}
+	}
+
+	private function disable_cron() {
+		$crons = _get_cron_array();
+
+		if ( empty( $crons ) ) {
+			return false ;
+		}
+
+		$hook = 'msm_cron_generate_sitemap_for_year_month_day';
+
+		foreach ( $crons as $timestamp => $cron ) {
+			foreach ( $cron[ $hook ] as $key => $data ) {
```

Comment:
This can cause an undefined index notice, yet it still seems to work. Just adding this note here so I remember to look into it and fix.